Update packages

KaiVolland opened the pull request:
This updates lots of packages to their latest version.

jansule reviewed (commented):
Thanks @KaiVolland! However, linting seems to fail for node v10.

KaiVolland commented:
> Thanks @KaiVolland! Looks good to me. However, linting seems to fail for node v10.

Node 10 is no longer maintained so i removed it from ci and added v16 instead.
Not sure if this is a breaking change then.
